src/templates/csr/public-good-test/index.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+	import { getUSPData, isCcpaOptedOut } from '$lib/cmp';
+	import { post } from '$lib/messenger';
+	import { getPageURL } from '$lib/messenger/get-page-url';
+	import { create, type PgmApiOptions } from '$lib/public-good';
+
+	let container: HTMLElement;
+
+	const refresh = () =>
+		post({
+			type: 'passback-refresh',
+			value: 'public-good',
+		});
+
+	const onload = async () => {
+		let url = await getPageURL();
+
+		const consentState = await getUSPData();
+
+		if (!consentState || isCcpaOptedOut(consentState)) {
+			!consentState && console.error('No consent state found');
+
+			return refresh();
+		}
```

Comment:
Very very minor but if `isCcpaOptedOut(consentState)` is `true` we have 'consent state' but consent is not given. So this might be a more accurate log message:

```suggestion
		if (!consentState || isCcpaOptedOut(consentState)) {
			!consentState && console.error('No consent');
			return refresh();
		}
```

Reply:
Yes you are right. The message has to be more accurate. Changed in 6bba7c362d5b1c5908f74f23c068358a3e334bb3